src/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+
+  margin: 0;
+}
+
+.block__red {
+
```

Comment:
do not need to add empty line at the start of style block

Reply:
but when I check input "npm run lint" it say me add emtpy line at the start of style block. and it say error when I don't that

Reply:
for example: src/style.css
  6:1  ×  Expected empty line before rule   rule-empty-line-before
 12:1  ×  Expected empty line before rule   rule-empty-line-before
 19:1  ×  Expected empty line before rule   rule-empty-line-before
 26:1  ×  Expected empty line before rule   rule-empty-line-before
 31:1  ×  Expected empty line before rule   rule-empty-line-before

Reply:
delete line